packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -20,11 +20,35 @@ export default function ({ types: t }) {
     }
   };
 
+  let propertyBuilders = {
+    spec: (ref, {key, value}) => t.expressionStatement(
+      t.callExpression(
+        t.memberExpression(
+          t.identifier("Object"),
+          t.identifier("defineProperty")
+        ),
+        [
+          ref,
+          t.isIdentifier(key) ? t.stringLiteral(key.name) : key, // TODO(motiz88): use key if computed
+          t.objectExpression([
+            // configurable is false by default
+            t.objectProperty(t.identifier("enumerable"), t.booleanLiteral(true)),
+            t.objectProperty(t.identifier("writable"), t.booleanLiteral(true)),
+          ].concat(value ? t.objectProperty(t.identifier("value"), value) : []))
+        ]
+      )
+    ),
+    loose: (ref, {key, value}) => t.expressionStatement(
+      t.assignmentExpression("=", t.memberExpression(ref, key), value) // TODO(motiz88): handle computed key
+    )
+  };
```

Comment:
You can also use template from the babel parameter - `export default function ({ types: t, template }) {` but it would be created inside the function